src/Mvc/src/Microsoft.AspNetCore.Mvc.NewtonsoftJson/Microsoft.AspNetCore.Mvc.NewtonsoftJson.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Internal.AspNetCore.Sdk">
+
+  <PropertyGroup>
+    <Description>ASP.NET Core MVC features that use Newtonsoft.Json. Includes formatter for JSON and JSON PATCH input and output, and extensions to Controllers.</Description>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <AssemblyName>Microsoft.AspNetCore.Mvc.Formatters.Json</AssemblyName>
```

Comment:
Keeping the assembly name so it's less breaky. Thoughts?

Reply:
I think we should lean into the breaking changes personally